README.md

```diff
@@ -28,10 +29,10 @@ Table of Contents
 
 ## Media Streaming
 
-  * https://plex.tv/ - Plex - Plex is a centralized home media playback system with a powerful central server. 
```

Comment:
It looks like my editor may have removed trailing spaces from a couple of lines.